packages/backend/src/api/endpoints/OrganizationApi.ts

```diff
@@ -206,13 +206,13 @@ export class OrganizationAPI extends AbstractAPI {
   }
 
   public async getOrganizationMembershipList(params: GetOrganizationMembershipListParams) {
-    const { organizationId, limit, offset } = params;
+    const { organizationId, limit, offset, orderBy } = params;
     this.requireId(organizationId);
 
     return this.request<PaginatedResourceResponse<OrganizationMembership[]>>({
       method: 'GET',
       path: joinPaths(basePath, organizationId, 'memberships'),
-      queryParams: { limit, offset },
+      queryParams: { limit, offset, orderBy },
```

Comment:
This goes with the other suggestion
```suggestion
      queryParams,
```

Reply:
@jacekradko Updated here